src/components/compound/DlInput/DlTextInput.vue

```diff
@@ -378,6 +381,10 @@ export default defineComponent({
         debounce: {
             type: Number,
             default: 100
+        },
+        disabledTooltip: {
+            type: String,
+            default: 'Disabled'
```

Comment:
Is there a requirement to have a disabled otoltip anyway ? If not, just make the default an empty string